### PR TITLE
Uploading photo

### PR DESCRIPTION
**Pull request by dxminikxx**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <script src="https://kit.fontawesome.com/7a6dd123f7.js" crossorigin="anonymous"></script>
```
Comment (novd) [inline public/index.html:6]: do we need this for icons?

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,118 @@
...
-  background-color: #282c34;
+body
+{
+  background: rgb(131, 195, 208);
+  display: flex;
+  align-items: center;
+  justify-content: center;
   min-height: 100vh;
+}
+.wrapped
```
Comment (novd) [inline src/App.css:14]: add new line before every new class

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,15 @@
...
-          Learn React
-        </a>
-      </header>
+      <PhotoUploader></PhotoUploader>
     </div>
   );
 }
 
 export default App;
+//znaleźć jak w js ładowanie zdj do aplikacji upload photo
```
Comment (novd) [inline src/App.js]: remove comment

File: src/script.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            <button onClick={() => setSelectedImage(null)}>Remove</button>
+          </div>
+        )}
+  
+       
+        <input
+          type="file"
+          name="myImage"
+          onChange={(event) => {
+            console.log(event.target.files[0]);
```
Comment (novd) [inline src/script.js]: remove console.log

File: src/shared/component/photo-uploader/PhotoUploader.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        </div>
+      </div>
+    );
+};
+
+ 
+export default PhotoUploader;
+
+
+//1. komponent photouploader ma pozwalac na wczytanie zdj i ma je wyświetlać
```
Comment (novd) [inline src/shared/component/photo-uploader/PhotoUploader.js]: remove comments

**Review by novd — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (dxminikxx) [reply public/index.html:6]: yes, it's required to use icons

File: src/App.css (hunk 2 above)
Comment (dxminikxx) [reply src/App.css:14]: done

File: src/App.js (hunk 3 above)
Comment (dxminikxx) [reply src/App.js]: done

File: src/script.js (hunk 4 above)
Comment (dxminikxx) [reply src/script.js]: done

File: src/shared/component/photo-uploader/PhotoUploader.js (hunk 5 above)
Comment (dxminikxx) [reply src/shared/component/photo-uploader/PhotoUploader.js]: done

**Review by novd — APPROVED** [review]

**Pull request merged**